octopusCats/src/main/scala/octopus/cats/package.scala

```diff
@@ -0,0 +1,23 @@
+package octopus
+
+import _root_.cats.data.{Validated, ValidatedNel, NonEmptyList}
```

Comment:
Nope. Package has the same name, root import is needed for the compiler to see the correct path.